daemon/services/controllers/array.go

```
@@ -26,6 +26,27 @@ func mdcmdExec(args ...string) error {
+			params["startState"] = state
+		} else {
+			logger.Warning("Array: could not read startState from var.ini; sending spin command without it")
+		}
+		return lib.EmhttpdRequest(params)
+	}
+	// Fallback: write directly to /proc/mdcmd (may fail on Unraid 7.3.x)
+	logger.Debug("Array: emhttpd socket not available, falling back to /proc/mdcmd for %s %s", mdcmdCmd, diskName)
+	return mdcmdExec(mdcmdCmd, diskName)
+}
+
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)